### ISSUE TITLE
Notifications are missing `isNew` and `read` fields

### ISSUE THREAD

**Issue by h1-the-swan** [markerikson/redux-quickstart-example-app#20]
The [tutorial](https://redux.js.org/tutorials/essentials/part-6-performance-normalization#showing-new-notifications) says: "Our fake API is already sending back the notification entries with `isNew` and `read` fields, so we can use those in our code." 

But the notifications sent from the server don't seem to have those fields: https://github.com/reduxjs/redux-essentials-example-app/blob/49fd0eab438e738bf5cb54431fd75dc4ae0fb817/src/api/server.js#L290-L301

**Comment by markerikson** [comment]
Yeah, I changed the API contents when I updated it to cover RTK Query, but I forgot to remove that line from the text.

**Comment by h1-the-swan** [comment]
OK. I'm having some strange unexpected behavior wrt these fields in this part of the tutorial. I thought it might be related to these missing fields, but I'm not sure it is.
